.github/scripts/python.sh

```diff
@@ -24,5 +24,5 @@ pip install -e .
 # Run GTSFM unit tests
 ##########################################################
 
-cd $GITHUB_WORKSPACE/tests
-python -m unittest discover
+cd $GITHUB_WORKSPACE
+pytest tests
```

Comment:
Yeah I think this is better as it will run all tests even when a failure occurs?

Reply:
Before we actually weren't running almost 100 tests -- we were only running the unittest ones, not the other pytest ones